sources/api/apiserver/src/server/exec/child.rs

```diff
@@ -241,6 +241,13 @@ impl ChildHandles {
     }
 }
 
+impl Drop for ChildHandles {
+    /// Ensures the RawFd is properly closed when ChildHandles is dropped.
+    fn drop(&mut self) {
+        let _ = close(self.pty_fd);
```

Comment:
nit: drop the `let` statement given that the output of close() is not really used?